config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -302,7 +302,7 @@ public OptionalLong getEcip1017EraRounds() {
 
   @Override
   public boolean isQuorum() {
-    return getOptionalBoolean("isQuorum").orElse(false);
+    return getOptionalBoolean("isquorum").orElse(false);
```

Comment:
Why was this changed?

Reply:
When it resolves the property is uses lowercase. In the genesis file it will still be `isQuorum`.